waiter/src/waiter/utils.clj

```diff
@@ -551,3 +551,24 @@
   (if (instance? ExceptionInfo e)
     (ex-info (.getMessage e) (update-fn (ex-data e)) (.getCause e))
     (ex-info (.getMessage e) (update-fn {}) (.getCause e))))
+
+(defn int-ceiling
+  "Gets the integer ceiling for a number."
+  [value]
+  (-> value
+      Math/ceil
+      int))
+
+(let [byte-array-type (Class/forName "[B")]
+  (defn byte-array?
+  "Tests if a value is a byte-array."
+  [val]
+    (when val
+      (instance? byte-array-type val))))
+
+(defn nanos->millis
```

Comment:
Should we have a PR in `clojure.time` for this?

Reply:
Removed this function.